Conceptpower+Spring/src/main/java/edu/asu/conceptpower/web/ConceptsMergeBeanValidator.java

```diff
@@ -0,0 +1,34 @@
+package edu.asu.conceptpower.web;
+
+import org.springframework.stereotype.Component;
+import org.springframework.validation.Errors;
+import org.springframework.validation.ValidationUtils;
+import org.springframework.validation.Validator;
+
+import edu.asu.conceptpower.app.bean.ConceptsMergeBean;
+
+@Component
+public class ConceptsMergeBeanValidator implements Validator {
+
+    @Override
+    public boolean supports(Class<?> clazz) {
+        return ConceptsMergeBean.class.isAssignableFrom(clazz);
+    }
+
+    @Override
+    public void validate(Object target, Errors errors) {
+        ValidationUtils.rejectIfEmptyOrWhitespace(errors, "word", "word", "Word cannot be empty.");
+        ValidationUtils.rejectIfEmptyOrWhitespace(errors, "selectedPosValue", "selectedPosValue",
+                "POS cannot be empty.");
+        ValidationUtils.rejectIfEmptyOrWhitespace(errors, "selectedListName", "selectedListName",
+                "Concept List cannot be empty.");
+        ValidationUtils.rejectIfEmptyOrWhitespace(errors, "selectedTypeId", "selectedTypeId",
+                "Concept Type cannot be empty.");
+
+        ConceptsMergeBean conceptMergeBean = (ConceptsMergeBean) target;
+        if (conceptMergeBean.getMergeIds() != null && conceptMergeBean.getMergeIds().size() < 2) {
```

Comment:
@jdamerow  MergedIds cannot be null. This is because mergedids are instantiated in ConceptMergeBean class. Do you still want me to check for the null condition? I am not sure if I am missing something here.